app/src/main/java/org/resolvetosavelives/red/widgets/Views.kt

```diff
@@ -8,6 +8,6 @@ fun EditText.showKeyboard() {
   post({
     this.requestFocus()
     val inputMethodManager = context.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
-    inputMethodManager.showSoftInput(this, InputMethodManager.SHOW_FORCED)
+    inputMethodManager.showSoftInput(this, InputMethodManager.SHOW_IMPLICIT)
```

Comment:
Why was this changed, exactly?

Reply:
I was initially using `SHOW_FORCED` because the keyboard wasn't showing up for one screen with `SHOW_IMPLICIT`. But I later saw that the keyboard never gets removed when the screen is changed.